### ISSUE TITLE
“<div class="u-baseline"></div>” garbage in examples

### ISSUE THREAD

**Issue by matthewpaulthomas** [vanilla-framework/vanilla-framework#1654]
1\. Go to any page in the Docs.
2\. Look at any of the code examples.

What you see:
```html
<div class="u-baseline-container u-hide">
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
    <div class="u-baseline"></div>
  </div>
```

What you should see: None of that.

**Comment by spencerbygraves** [comment]
This seems to be a problem in all examples.

**Comment by Caleb-Ellis** [comment]
I think this might be the straw that broke the camel's back for example.js. I'll work on a naive solution as a temp fix

**Comment by Caleb-Ellis** [comment]
Fixed as of #1657
